issue_06/script.js

```diff
@@ -11,21 +11,27 @@ const contents = [{
 }];
 
 const ul = document.getElementById('js-ul');
-const checkContents = new Promise((resolve) => {
-    resolve(contents);
-});
-checkContents.then((value) => {
+const wait = (sec) => {
+    return new Promise((resolve) => {
+        setTimeout(resolve, sec * 1000);
+    });
+}
```

Comment:
MUST
Resolve without receiving the contents.

Reply:
The process simply starts after 3 seconds, but the "contents" was not passed.
Fixed!
https://github.com/yoshikawa-akr/js-lessons/pull/6/commits/5e2cb0adcc2df2601b0b852975d3232d81cd26c5